src/components/common/animations/Base.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+class AnimationBase extends React.Component {
```

Comment:
inheritance in prohibited in React, composition is preferred
as the quick solution for etoro it's ok but generally we should
create separated component running animations (or, better - custom hook)
and use it instead